mod_reforged/hooks/config/strings.nut

```diff
@@ -1554,11 +1554,11 @@ foreach (vanillaDesc in vanillaDescriptions)
 		}]
  	}),
 	RF_Professional = ::UPD.getDescription({
- 		Fluff = "You\'re a professional, experienced fighter, able to wield many weapons in many styles!",
+ 		Fluff = "You\'re a professional, experienced fighter!",
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
```

Comment:
This must now be `OneTimeEffect`

Reply:
If the perk is refundable, should it still be OneTimeEffect? Or does it make more sense to keep it "Passive" in that case? I think the latter is more sensible because it passively grants you the other 2 perks but you lose those perks when you lose this perk.

Reply:
You decide.
Overall I prefer making this perk non-refundable in which case this question is easy. But in this case were it is redunable, I have no idea and don't care.